portal-bridge/src/types/full_header.rs

```diff
@@ -162,6 +162,23 @@ mod tests {
         block_body.validate_against_header(&header).unwrap();
     }
 
+    #[rstest::rstest]
+    // without blob txs
+    #[case::legacy("block_19433902_value.json")]
+    // with blob txs
+    #[case::legacy("block_19433903_value.json")]
```

Comment:
Not sure what happened, but it seems like these should be `"../test_assets/mainnet/block_17034873_value.json"` ...
or you could do `format!(""../test_assets/mainnet/block_{}_value.json", block_number)` below